lib/github-glue.ts

```diff
@@ -80,9 +80,9 @@ export class GitHubGlue {
      */
     public async addPRComment(pullRequestURL: string, comment: string):
         Promise<{id: number, url: string}> {
-        await this.ensureAuthenticated();
         const [owner, repo, nr] =
             GitGitGadget.parsePullRequestURL(pullRequestURL);
```

Comment:
For future consideration, change `pullRequestURL` to an object with `[owner, repo, nr]`.  There seems to be a lot of reparsing of the url.  I understand the original script did not have the benefit of objects.

Reply:
Good point. I opened https://github.com/gitgitgadget/gitgitgadget/issues/150 as a reminder.